cabify-solution/src/clients/saveMessage.js

```diff
@@ -1,13 +1,17 @@
-import Message from "../models/message.js";
+import { Message, MessageCopy } from "../models/message.js";
+
 
 export default async (messageParams) => {
   const message = new Message(messageParams);
-  
+  const messageCopy = new MessageCopy(messageParams)
+
   try {
     const doc = await message.save();
+    const doc2 = await messageCopy.save()
 
     console.log("Message saved succesfully:", doc);
-    return doc;
+    console.log("Messages saved succesfully in Copy Database", doc2)
```

Comment:
Put the logs immediately after the operation so we can be sure that everything happened as should.

const doc = .....
console.log()
const doc2 = .....
console.log()